src/DonateTo.WebApplication/ClientApp/src/app/admin/priority-questions/question-create/questions-create.component.html

```
@@ -248,50 +248,46 @@ <h4 class="section-title">{{ 'Admin.PriorityQuestion.Create.Title' | translate }
-                <td>{{ item.controlType?.name }}</td>
-                <td>{{ item.defaultValue }}</td>
-                <td>
-                  <span *ngFor="let option of item.options"> {{ option.label }} -</span>
-                </td>
-                <td>
-                  <a (click)="editQuestion(item)">{{ 'Admin.PriorityQuestion.Table.EditQuestion' | translate }}</a>
-                  <nz-divider nzType="vertical"></nz-divider>
-                  <a (click)="removeQuestion(item)">{{ 'Admin.PriorityQuestion.Table.DeleteQuestion' | translate }}</a>
-                </td>
-              </tr>
-            </tbody>
-          </nz-table>
-        </div>
-      </nz-form-control>
-    </nz-form-item>
   </form>
 
+  <span class="item-required">{{ 'Admin.PriorityQuestion.Form.PriorityQuestionRequiredError' | translate }}</span>
+  <br />
+  <nz-table
+    [nzData]="questions"
+    [nzShowPagination]="false"
+    [nzFrontPagination]="false"
+    [nzShowPagination]="false"
+    [nzNoResult]="customTpl"
+  >
+    <thead>
+      <tr>
+        <th *ngFor="let column of listOfColumns">
+          {{ column.name | translate }}
+        </th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr *ngFor="let item of questions">
+        <td>{{ item.label }}</td>
+        <td>{{ item.placeholder }}</td>
+        <td>{{ item.order }}</td>
+        <td>{{ item.weight }}</td>
+        <td>{{ item.controlType?.name }}</td>
+        <td>{{ item.defaultValue }}</td>
+        <td>
+          <span *ngFor="let option of item.options"> {{ option.label }} -</span>
+        </td>
+        <td>
+          <a (click)="editQuestion(item)">{{ 'Admin.PriorityQuestion.Table.EditQuestion' | translate }}</a>
+          <nz-divider nzType="vertical"></nz-divider>
+          <a (click)="removeQuestion(item)">{{ 'Admin.PriorityQuestion.Table.DeleteQuestion' | translate }}</a>
+        </td>
+      </tr>
+    </tbody>
+  </nz-table>
+  <div id="weightError" hidden="true" class="error-required">
```

You can use *ngIf="!isValid" here to show/hide the error instead of doing it on the .ts file

---

better solution thanks